deploy/schemas/sample-backup-configs/sample_config_build.yaml

```diff
@@ -0,0 +1,10 @@
+build_farm:
+  base_recipe: str()
+  # map values unspecified since these are build farm specific
```

Comment:
Ok the way I hacked this up (which I'm not sure if it is too much of a hack), is that I parse these files in 2 runs.

Run 1 - Parse the file pointing to an extra schema file that has a generic `map()` validator (aka same as before). This pass is mainly so that we can catch issues outside of the overrides section.

Run 2 - Parse the file and point to an extra schema file that describes the run/build farm recipe (based on the `base_recipe` K/V). This pass re-checks the other K/V pairs outside of the overrides but also checks the overrides as well. Note that if Run 1 fails, this run will never happen so this is basically just checking overrides.

Reply:
I solved this. Basically, I parse the overrides as if it is a recipe file itself by regenerating the YAML file from the dict of override args.